README.md

```diff
@@ -111,13 +94,43 @@ Amplify requires Xcode 13.4 or higher to build.
     ```swift
     import Amplify
 
-    // ... later
+    // ...
 
-    func doUpload() {
-        Amplify.Storage.uploadFile(...)
+    func signIn() async throws {
+        let signInResult = try await Amplify.Auth.signIn(...)
```

Comment:
`Storage.uploadFile(key:local:)` is an atypical API due to progress reporting. Opting to include a standard and commonly-used API to show async/await.